### PR TITLE
276 add combined building loss for wind wave and surge

### PR DESCRIPTION
**Pull request by navarroc**
This analysis uses the 3 damage outputs and computes the structural and content loss from the different hazards. 

I think this analysis can be improved to handle the case where you only provide one hazard and it can give you the damage to that component since this analysis is breaking down which component is impacted by which hazard (e.g. Flood is the only hazard that is needed for content loss).

### CODE REVIEW

File: tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    start = timer()
+
+    # Combined building damage to determine maximum damage state
+    combined_bldg_loss = CombinedWindWaveSurgeBuildingLoss(client)
+    combined_bldg_loss.load_remote_input_dataset("buildings", bldg_dataset_id)
+    combined_bldg_loss.set_input_dataset("surge_wave_damage", surge_wave_damage)
+    combined_bldg_loss.set_input_dataset("wind_damage", wind_damage)
+    combined_bldg_loss.set_input_dataset("flood_damage", flood_damage)
+    combined_bldg_loss.load_remote_input_dataset("structural_cost", "63fd15716d3b2a308ba914c8")
+    combined_bldg_loss.load_remote_input_dataset("content_cost", "63fd16956d3b2a308ba9269a")
```
Comment (longshuicy) [inline tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py:79]: We would need to release these two datasets?

File: pyincore/analyses/combinedwindwavesurgebuildingloss/combinedwindwavesurgebuildingloss.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+        loss_df['ff_loss'] = new_combined_df.apply(lambda row: np.dot(row[['sw_DS_0', 'sw_DS_1', 'sw_DS_2',
+                                                                           'sw_DS_3']], np.array([0.1, 0.5, 0.75, 1])) *
+                                                   structure_cost.loc[int(row['arch_flood'] - 1),
+                                                   "Flooring and Foundation"], axis=1)
+
+        # Computes frame loss from the dominant hazard between wind and surge-wave intensities
+        def compute_frame_loss(row):
+            if row['w_DS_3'] >= row['sw_DS_3']:
+                return np.dot(row[['w_DS_0', 'w_DS_1', 'w_DS_2', 'w_DS_3']],np.array([0.25, 0.5, 0.75, 1])) * \
```
Comment (longshuicy) [inline pyincore/analyses/combinedwindwavesurgebuildingloss/combinedwindwavesurgebuildingloss.py:117]: Are those number fixed? 0.25, 0.5 .. i guess we are dividing 1/4 cuz we have 4 ds..

**Review by longshuicy — APPROVED** [review]

File: tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    f_bldg_dmg.set_parameter("hazard_id", hazard_id)
+    f_bldg_dmg.set_parameter("num_cpu", 4)
+    f_bldg_dmg.run_analysis()
+
+    # Get damage outputs from different hazards
+    surge_wave_damage = sw_bldg_dmg.get_output_dataset("ds_result")
+    wind_damage = w_bldg_dmg.get_output_dataset("ds_result")
+    flood_damage = f_bldg_dmg.get_output_dataset("ds_result")
+
+    print("start timer")
```
Comment (Vismayak) [inline tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py:69]: Do we need to print the timer starting, I do not see it being done in any other tests...

Comment (Vismayak) [inline tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py:69]: Do we need to print the timer starting, I do not see it being done in any other tests...

File: tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py (hunk 1 above)
Comment (navarroc) [reply tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py:79]: Yes

File: pyincore/analyses/combinedwindwavesurgebuildingloss/combinedwindwavesurgebuildingloss.py (hunk 2 above)
Comment (navarroc) [reply pyincore/analyses/combinedwindwavesurgebuildingloss/combinedwindwavesurgebuildingloss.py:117]: Yes, they were provided by Jiate. Maybe next release we can see if those should be inputs or if they are fixed.

File: tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py (hunk 3 above)
Comment (navarroc) [reply tests/pyincore/analyses/combinedwindwavesurgebuildingloss/test_combinedwindwavesurgebuildingloss.py:69]: Let me remove that.

**Review by Vismayak — APPROVED** [review]
